gecko/crawlers/doxygen.py

```diff
@@ -0,0 +1,284 @@
+import os
+import re
+from urllib.parse import urljoin, urlparse
+
+from bs4 import BeautifulSoup, Tag
+
+from .crawler import Crawler
+
+
+def sanitize_title(title):
+    title = re.sub(r'\s+', ' ', title).strip()
+
+    return title
+
+
+def sanitize_content(content):
```

Comment:
Sanitizing content is unnecessary since it will be converted into a search index where generic words like 'more,' 'it,' and 'there' won't have much impact.

Reply:
Can we skip them entirely since we already have them on their dedicated page? I mean if we want to send someone to a related section in the docs it would be better to show the dedicated page.

Reply:
Oh, that's an interesting idea. What about header file entries?

Here is an example:
https://www.boost.org/doc/libs/1_82_0/libs/hana/doc/html/concept_2applicative_8hpp.html

It currently has three entries:
- one for the file itself, with `More...`,
- one for `Namespaces`,
- one for `Detailed Description`.

What do we want to keep?

Reply:
I've just removed the tables with `More...`  (with a very simple method) so only those header files are left.

Reply:
> one for the file itself, with More...,

I don't know how useful this section is, because it just repeats what already is in the Detailed Description.